google/cloud/storage/tests/read_object_stall_regression/read_object_stall_test.cc

```diff
@@ -336,17 +340,22 @@ class ReadObjectStallTest
                 << std::flush;
     }
     std::geometric_distribution<int> sleep_period(1.0 / flag_average_delay);
+    std::uniform_real_distribution<double> use_long_delay(0.0, 1.0);
     for (int i = 0; i != max_chunks; ++i) {
-      int sleep_seconds = sleep_period(generator_);
-      std::cout << "Sleeping for " << sleep_seconds << "s\n" << std::flush;
-      std::this_thread::sleep_for(std::chrono::seconds(sleep_seconds));
-
+      auto sleep_seconds = Delay();
+      std::cout << "Sleeping for " << sleep_seconds.count() << "s\n"
+                << std::flush;
```

Comment:
Do we need the flush after a '\n'?

Reply:
Probably, the next line is a sleep. Without the flush one may wonder why is the program making no progress.  :man_shrugging:  typically I just read the logs after the program runs for a couple of hours.

Reply:
If we're trying to read block-buffered logs while they are still open, I agree there is an issue.  Perhaps we should try to line buffer cout/clog no matter where they are directed?